Auth.php

```diff
@@ -678,34 +674,43 @@ protected function addUser($email, $password, $params = [], &$use_email_activati
         }
 
         $password = $this->getHash($password);
+        if(strlen($this->config->table_params)==0){ 
```

Comment:
I prefer `empty(this->config->table_params)` (strlen is slow (performance))

Reply:
There was a reaaaaly strange situation when I used empty $this I kept getting affirmative assertion, so I ended with the sterlen is slow :(... I could change them to strcmp($this->config->table_params, '') maybe?